### PR TITLE
improve the segment-anything example

### PR DESCRIPTION
**Pull request by guschmue**

### CODE REVIEW

File: js/segment-anything/webpack.config.js [hunk 1]
```diff
@@ -1,27 +1,52 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT license
+import CopyWebpackPlugin from 'copy-webpack-plugin';
```
Comment (satyajandhyala) [inline js/segment-anything/webpack.config.js:1]: Is copyright removed intentionally?

**Comment by fs-eire** [comment]
maybe not using any bundler is a better idea. this helps to remove those unnecessary files (webpack.conf.js, package.json) with upcoming security warnings.

File: js/segment-anything/webpack.config.js (hunk 1 above)
Comment (guschmue) [reply js/segment-anything/webpack.config.js:1]: just a std webpack.config

**Comment by guschmue** [comment]
Going to add support for slimsam soon which will need some extra processing at build time so very soon I'll need at least package.json.

**Review by fs-eire — APPROVED** [review]

**Pull request merged**